src/components/Purchase/CheckoutPage.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+import style from './CheckoutPage.module.css';
+import CheckoutForm from './CheckoutForm';
+
+// Make sure to call loadStripe outside of a component’s render to avoid
+// recreating the Stripe object on every render.
+// This is your test publishable API key.
+const stripePromise = loadStripe(
+  'pk_test_51Nfj2pKnb0YvaiB3W0z31bbB7OPEnqem6wnpSbohudeiDkj1NUde9M5kgUwozzrfBqGqRpWU5ivDIInzWdt4q5zw00vMz7fc1c'
+);
+//Temporary variables for testing
+//const paymentAmount = 1099; // This is $10.99 payment amount(added for testing) This amount variable should came from cart
+const album = '876r75675';
```

Comment:
This is currently causing an error because its an invalid id for mongo (too short to be a valid Mongo ObjectId).

We can grab one of the existing album ids from `localhost:8000/api/v1/albums`  and use that as a placeholder here:

```suggestion
const album = '64d2a94c793389a43fc5a8d6';
```

Reply:
I successfully updated the album ID.